src/landingPage.scss

```diff
@@ -103,6 +128,12 @@
         margin-left: 15px;
     }
 
+    .supporters{
+        display: flex;
+       
```

Comment:
nit: extra line here and generally all over this file

Reply:
I went through the file and tried to remove any extra lines!